server/src/configs/validators.rs

```diff
@@ -77,8 +77,8 @@ impl Validatable<ServerError> for SegmentConfig {
 impl Validatable<ServerError> for MessageSaverConfig {
     fn validate(&self) -> Result<(), ServerError> {
         if self.interval == 0 {
-            // TODO: use a custom error
-            panic!("Message saver interval must be greater than 0.")
+            error!("Saver Interval size cannot be zero, it must be greater than 0.");
```

Comment:
Let's keep the `Message saver` name instead of just `Saver` and the interval without the capital I :)